libs/core/src/main/java/org/opensearch/BaseOpenSearchException.java

```diff
@@ -440,19 +509,86 @@ private String getShardIdString() {
     }
 
     /**
-     * An ExceptionHandle for registering Exceptions that can be serialized over the transport wire
+     * Returns an array of all registered handle IDs. These are the IDs for every registered
+     * exception.
+     *
+     * @return an array of all registered handle IDs
+     */
+    static int[] ids() {
```

Comment:
Why need to duplicate that vs calling `OpenSearchExceptionHandleRegistry.ids()`?